1. Unit testing/1. Basics/Task 02 - Exam conflicts/examConflicts.test.ts

```diff
@@ -0,0 +1,182 @@
+import { describe, expect, it } from "vitest";
+import {
+  canRegister,
+  computeRegistrationFee,
+  detectExamConflicts,
+  Exam,
+  ExamRaw,
+  parseExamSchedule,
+  scheduleExamReminders,
+} from "./examConflicts";
+
+const examRaw: ExamRaw = {
+  subject: "Networking",
+  date: "2025-06-11T15:00:00Z",
+  durationMinutes: 90,
+  location: "205 C-1",
+  fee: 20,
+  earlyBirdDeadline: "2025-06-05T00:00:00Z",
+  registrationDeadline: "2025-06-10T09:00:00Z",
+};
```

Comment:
I like how tidy the code is, thanks to using just this example exam and modifying it for the tests